locale/ru/docs/guides/getting-started-guide.md

```diff
@@ -25,4 +25,4 @@ server.listen(port, hostname, () => {
 });
 ```
 
-After that, run your web server using ``` node app.js ```, visit http://localhost:3000, and you will see a message 'Hello World'
+После этого запустите ваш веб-сервер используя комманду  ``` node app.js ```, откройте http://localhost:3000 в браузере, и вы увидите сообщение 'Hello World'
```

Comment:
`комманду` -> `команду`

Reply:
Merci! I will read thorugh further translations I'm working on more carefully!
@indutny @alexandrtovmach 
Also, can we connect somewere, so I can request for a fact check in my translations prior pull requests? I find some parts of English guides have non deterministic meaning. Thus it would be nice to check up the semantics with some one trully experienced with the platform.